client/src/components/GameBoxScore.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import { useMediaQuerySSR } from 'hooks'
+import { VStack, HStack, Table, Thead, Tbody, Tr, Th, Td, Box } from '@chakra-ui/react'
+
+const Totals = ({ playersStats }) => {
+    var point_total = 0
+
+    for (let i = 0; i < playersStats.length; i++) {
+        point_total += playersStats[i].points
+    }
+
+    return ( 
+        <Tr borderTop="2px solid grey" fontWeight="semibold" fontSize="1.1rem">
+            <Td>TOTALS</Td>
+            <Td paddingLeft="1.75rem">{point_total}</Td>
+        </Tr>
+     )
+}
+
+const PlayerData = ({ playerStat }) => {
+    return ( 
+        <Tr fontSize="1.1rem">
+            <Td>{playerStat.playerId.name}</Td>
+            <Td paddingLeft="1.75rem">{playerStat?.points >= 0 ? playerStat?.points : "-"}</Td>
+        </Tr>
+    )
+}
+
+const PlayersTable = ({ playersStats, stats_fields }) => (
+    <Box
+            h="100%"
+            w="100%"
+            bg="white"
+            borderRadius="1rem"
+            boxShadow="0 5px 10px rgba(0,0,0,0.1), 0 3px 3px rgba(0,0,0,0.12);"
+            padding="1rem"
+    >
+    <Table variant="striped" size="sm">
+        <Thead>
+            <Tr borderBottom="2px solid grey" fontWeight="semibold">
+                <Th minW="100px">PLAYER</Th>
+                {stats_fields.map((s) => <Th key={s}>{s.toUpperCase()}</Th>)}
+            </Tr>
+        </Thead>
+        <Tbody>
+            {playersStats.map((playerStat) => (
+                <PlayerData playerStat={playerStat}/>
```

Comment:
Might be wrong, (can double check this in console logs) but would be nice to have a `key` property here to get rid of the annoying log messages. id can probably be `playerStat._id`

Reply:
done! thank you~~